redis/vibe/db/redis/redis.d

```diff
@@ -33,6 +33,7 @@ import std.utf;
 */
 RedisClient connectRedis(string host, ushort port = RedisClient.defaultPort)
 {
+	assert(!host.startsWith("redis://", "Host must be a valid IP or domain, but not an URI."));
```

Comment:
We could also think about parsing the `host` URL if the user accidentally called the wrong function, but imho it's better to use an error message here, because if the user has a full redis URI, he should have called `connectRedisDB` which returns a `RedisDatabase` and not only a `RedisClient`